### ISSUE TITLE
Drug accumulating in intracellular compartment of brain

### ISSUE THREAD

**Issue by jhhughes256** [Open-Systems-Pharmacology/Forum#145]
Hi,

I have been developing a model in mice using PKSim, where the model shows the drug accumulates in the brain, while my data does not. Below I've included a plot showing the arterial blood concentrations and tissue concentrations for the brain, liver, lung and kidney. Each of the concentration time curves follow the changes seen in arterial blood, however I am having trouble explaining the lack of changes in the brain.
![total tissue conc pksim](https://user-images.githubusercontent.com/20290321/36763146-9d4fdb42-1c76-11e8-93fd-41eb0933ace6.png)

Looking more closely I can see that it is accumulating specifically in the intracellular compartment of the brain. It takes up the majority of the volume of the brain and as a result the concentrations peak later, but the fact the concentrations are so flat, and followed by drug leaving the intracellular space at a very low rate is something I'm having difficulty understanding. Below I have a plot showing the total concentrations in the brain tissue (blue), and concentrations in plasma (brown), blood cells (green), interstitial (grey) and intercellular space (mustard). The blue dots represent the dose normalised data I have for the drug concentrations in mouse brain tissue.
![brain conc pksim](https://user-images.githubusercontent.com/20290321/36763151-9fd0868c-1c76-11e8-9c34-84d111459ab2.png)

The molecule I am using isn't particularly lipophillic `logP = -0.4`, is a small molecule `MW = 259.26` and has a fraction unbound of ~70%. I have begun to look through the passive transport equations in MoBi but have had difficulty finding why this is occurring. I am unsure if I have missed an equation describing the blood brain barrier, but can't really find anything that explains what I'm seeing. 

Cheers, Jim

**Comment by PavelBal** [comment]
Hi Jim,

it is an interesting question. I can reproduce this behavior with a dummy compound using the parameters you mentioned. I took a look at the equations in MoBi and can at least partly understand, how the blood-brain barrier (BBB) is implemented.

The first part of BBB takes place in the diffusion of molecules from blood plasma into interstitial space, modeled by the passive transport `PassiveDiffusionPl2Int`. The transport rate is given by the equation

`fu * P_endothelial * SA * (C_pls - C_int / K_int_pls)`

The parameter `P_endothelial` describes the permeability of the molecule through the endothelial barrier. For all tissues but the brain, its value is set to arbitrary 100 cm/min, implying instantaneous equilibration between plasma and interstitial space. For the brain, its value is calculated as `P/2` with `P` being the permeability of the molecule across a cell membrane (lipid bi-layer), calculated from its molecular weight and lipophilicity. `P` is divided by 2 with the assumption that the molecule has to pass two lipid bi-layers on its way through endothelium.

In summary, the first part of the BBB is the restriction of extravascularisation to simple diffusion process, while in other tissues the extravascularisation is assumed to be rapid (instantaneous).

The second part is the diffusion of the molecules from interstitial into intracellular space, given by the passive transport `PassiveDiffusionInt2Cell`. The rate of the process is given by the equation 

`K_water_int * PA_int_cell * C_int - K_water_cell * PA_cell_int * C_cell`

The parameters `PA_int_cell` and `PA_cell_int` have the same values, and are defined by the parameter `Surface area (interstitial/intracellular)`, located at the neighborhood `Brain_int_Brain_cell` (substitute "Brain" for other organs). Now, the calculated surface area for the brain is five orders of magnitude smaller than the calculated surface area for the liver (988 vs. 49633653 cm²), and organ with the comparable volume , resulting in extremly low exchange rate between interstitial and intracellular.

`Surface area (interstitial/intracellular)` for the brain is given by the equation 

`(V_brn  * 1000 / 1.671)^0.75 * 0.0006 * 1E2`

with V_bnr the volume of the brain, and the same parameter for the liver is given by 

`(V_liv * 1000 / 10)^0.75 * 82 * 1E2`

I have no idea where these values come from.

Because the concentration difference between interstitial and intracellular is huge at the beginning of the simulation, the compound is transferred into the cells. Later on, the concentration gradient is reversed (higher conc. in the cells), but is too small that the calculated exchange rate would be of any significance. This leads to the "accumulation" of the compound in the cells.

So while we understand the technical part, how this happens, I am still nor sure why it is implemented this way. Your data indicates that no accumulation in the brain takes place. No we can speculate, which part of the implemented BBB is causing this inconsistence. Either the extravascularisation (plasma -> int) is too fast, or the exchange between int -> cell is too slow. OR there are some active transport processes involved... I am not an expert on the BBB, so I would like to hear oppinions from people having more experience and understanding of the transport processes taking place in the brain.

**Comment by StephanSchaller** [comment]
Taking endothelial surface area out of the discussion (this value is most likely correct), what is not accounted for is active transport and brain "metabolism".
Also judging from your plot I wouldn't call it accumulation but drug retention in the brain.
Maybe any of the following publications are of help:
http://jpet.aspetjournals.org/content/313/3/1254
http://onlinelibrary.wiley.com/doi/10.1002/9781118788523.ch14/summary
https://www.ncbi.nlm.nih.gov/pubmed/24647103
https://www.ncbi.nlm.nih.gov/pubmed/26869440
https://www.ncbi.nlm.nih.gov/pubmed/22864977
https://www.ncbi.nlm.nih.gov/pubmed/23784110
https://www.ncbi.nlm.nih.gov/pubmed/27236639

... let us know what you find...!

**Comment by jhhughes256** [comment]
@PavelBal Thanks for finding those equations. I feel like I was so close to finding them, but I am still learning how to dig through all the good stuff in MoBi. 
Also thanks to others for the articles. I am familiar with those on lenalidomide, but getting some knowledge on PBPK models for brain distribution should definitely get me thinking the right way to tackle this problem!

I think describing active transport processes is easily my best bet at the moment, hopefully despite lenalidomide being a weak substrate I can coax something compelling out of PKSim.

**Comment by StephanSchaller** [comment]
Keep us posted!

**Comment by jhhughes256** [comment]
Well it's taken me a while to work it out, but have tried out active transports and brain metabolism.

__Active Transports__
Active transports were easy to rationalise. As @tobiasK2001 has mentioned, lenalidomide is a weak substrate of p-gp and so having an active transport at the BBB to represent that makes sense. However when I do that I still maintain this strange retention of drug in the brain. 
![secrpgp](https://user-images.githubusercontent.com/20290321/37078746-596fe61e-2230-11e8-81c3-448e3cd18461.png)
In the above image tissue (blue), plasma (aqua), blood cell (orange), interstitial (pink) and intracellular (green) concentrations in the brain are plotted. The blue dots represent observed brain data. 

I added a transport protein to my mouse building block with expression values taken from [here](https://www.ncbi.nlm.nih.gov/gene/18671). I then included it in my molecule building block as a Specific Active Transport that follows Michaelis-Menten type kinetics. The parameters used to make the plot are `Transporter conc = 0.67` `Vmax=10.68` `Km=52.35`. The Km is quite high so it seems the process is fairly linear within the data that I have. Going in to MoBi, the transports seem to be where I want them to be, (at the blood brain barrier: interface of Brain Plasma and Brain Interstitial; in the kidney: interface of Kidney Intracellular and Urine) so I believe I've manage to implement this correctly...

You might notice my data has changed a little, worked out how to use parameter identification without using dose-normalised data, so it's no longer transformed. (Also got around to adding standard deviations to my observed data).

__Brain Metabolism__
Brain metabolism can be done by CYP450's but lenalidomide isn't a substrate of any CYP450s. It undergoes majorly renal excretion with 84% being collected in urine unmetabolised. The other 16% is presumable metabolised via hydrolysis. It is unclear what does the metabolism... one paper refers to some unpublished data they have where lena had a half life of 24hr in buffer and 8hr in plasma. 

So I set up a "hydrolysis enzyme" in plasma and tweaked it so the simulation had basic renal clearance making up 84% with the rest being attributed to the hydrolysis. Then I set up a second "hydrolysis enzyme" in the tissues and used parameter identification and let the hydrolysis in tissues and plasma be identified (with the hydrolysis parameters that give 16% hydrolysis being the upper boundary) to see if I could tease anything out. It resulted in the same flat profile with a bit more of an incline, but doesn't really match my data.

I also set up a parameter identification just using my brain data, and removed boundaries to see what was required to get something looking like the data. While I get a slope that "agrees" with my data, it still misses the Cmax by almost 10-fold, and my plasma data is no longer matching up well.
![hydropgp](https://user-images.githubusercontent.com/20290321/37078731-4f853c08-2230-11e8-8084-49ab9e07912b.png)
In the above image image tissue (blue), plasma (aqua), blood cell (orange), interstitial (pink) and intracellular (green) concentrations in the brain are plotted. The blue dots represent observed brain data, red dots observed plasma data. 

The issue with using parameter identification on just the brain data is that I end up with more metabolised lena in the urine than unmetabolised, so probably not a great representation of what is happening.

Still working out where to go from here, will post again if I work anything else out!

**Comment by tobiasK2001** [comment]
Dear Jim,
thans for the update. Indeed inserting the transporter alone does not help. And your interstitial conc looks a bit strange. Just for checking, did you set the apical/basolateral orintation like this:
![grafik](https://user-images.githubusercontent.com/25585248/37084235-393813d2-21f2-11e8-8721-abadab7f05ec.png)
Further you might try to increase the brain interstitial <-> intracellular surface area like e.g. 100 x like this:
![grafik](https://user-images.githubusercontent.com/25585248/37084349-8d1283fc-21f2-11e8-8d47-c08e979ce93e.png)
And than see what is happening to your slope.
From your investigation so far it seems to me that metabolism cannot explain it all alone. Maybe if you try a combination of metabolism (informed by your metabolite in urine (and if you have in plasma), Transport (informed by fraction parent excreted in urine) and an increased  brain interstitial <-> intracellular surface the PI might come to a solution...
Good luck and keep us posted ;-)
Tobias

**Comment by jhhughes256** [comment]
Hi @tobiasK2001 ,

My apical/basolateral orientation is as your screenshot describes. I agree that the interstitial concentrations are strange, the behaviour makes me doubt that it is working as I think it should, but from what I could see in MoBi I don't believe I've done anything strange.

I tried out your suggestion of increasing the surface area of the interface between interstitial and intracellular compartments of the brain. I also created a fraction excreted dataset to try to get PI to give parameters that resulted in a fraction excreted in the urine unchanged closer to 0.84.
![fractionexcreted](https://user-images.githubusercontent.com/20290321/37190374-53f8320c-23a9-11e8-9adc-abbd334934ab.png)
In the above image image tissue (blue), plasma (aqua), blood cell (orange), interstitial (pink) and intracellular (green) concentrations in the brain are plotted. The red line represents the venous plasma concentrations. The blue dots represent observed brain data, red dots observed plasma data. The dashed purple line represents the fraction excreted in the urine unchanged, with the purple dots being the created dataset with five points at 0.84 fraction excreted. This is the reference value for humans though, and could be different in mice.

It certainly causes the two compartments to equilibrate more rapidly, but I'm not sure if it solves my underlying problems of retention. Will keep this thread updated with any further things I work out over the weekend.

Cheers, Jim
